pkg/pgxconn/pgx_conn.go

```diff
@@ -70,3 +136,15 @@ func (p *connImpl) NewBatch() PgxBatch {
 func (p *connImpl) SendBatch(ctx context.Context, b PgxBatch) (pgx.BatchResults, error) {
 	return p.Conn.SendBatch(ctx, b.(*pgx.Batch)), nil
 }
+
+// filters out indentation characters from the
+// SQL query for better query logging
+func filterIndentChars(query string) string {
+	dropChars := []string{"\n", "\t", "\""}
```

Comment:
it's probably better to use a replacer here https://golang.org/pkg/strings/#Replacer